src/main/java/com/plivo/api/models/address/Address.java

```diff
@@ -0,0 +1,181 @@
+package com.plivo.api.models.address;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.plivo.api.models.base.BaseResource;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Remove NON_NULL, 
It would better to include null field show signature will be consistent.

Reply:
as discussed submitted a PR for the same